rmc/utils/mpc.py

```diff
@@ -302,9 +280,8 @@ def create_context_with_oe(
     ht = ht.select(oe_col).key_by("locus", "alleles")
     ht = ht.collect_by_key()
     ht = ht.annotate(**{oe_col: hl.nanmin(ht.values[oe_col])})
-    transcript_col = "transcript_rescue" if include_rescue else "transcript"
     ht = ht.annotate(
-        **{transcript_col: ht.values.find(lambda x: x[oe_col] == ht[oe_col]).transcript}
+        **{"transcript": ht.values.find(lambda x: x[oe_col] == ht[oe_col]).transcript}
     )
     ht = ht.drop("values")
 
```

Comment:
Can't comment but the section on lines 288-302 hasn't yet been edited to remove the joining code from the rescue PR: